.gitignore

```diff
@@ -1,3 +1,4 @@
+.bundle
```

Comment:
FYI if you put that in your global .gitignore you neer have to add it to another project